### PR TITLE
Reduce fetch-retry number of tries to 3

### PR DESCRIPTION
**Pull request by mvaivre**
Works based on my initial testing but needs to be tested with huge addresses.

### CODE REVIEW

**Review by nop33 — APPROVED** [review]
I don't know any big addresses, do you have any available? The cool thing with the explorer is that we can test with mainnet too. I'm approving it cause code-related looks good.

**Comment by mvaivre** [comment]
@nop33 

testnet: [18x1bYjxKSqJcnCjHv47TU49RAP2zn1PTVYW1J3py8UHc](https://testnet.alephium.org/addresses/18x1bYjxKSqJcnCjHv47TU49RAP2zn1PTVYW1J3py8UHc)
[1DrDyTr9RpRsQnDnXo2YRiPzPW4ooHX5LLoqXrqfMrpQH](https://testnet.alephium.org/addresses/1DrDyTr9RpRsQnDnXo2YRiPzPW4ooHX5LLoqXrqfMrpQH)
[19qG4ztYpfoxvzmPsnJfQ1QByGqWTVtrW4Vz3h3GbQmVT](https://testnet.alephium.org/addresses/19qG4ztYpfoxvzmPsnJfQ1QByGqWTVtrW4Vz3h3GbQmVT)

**Review by nop33 — APPROVED** [review]
Ι've tested all those addresses and it looks good! Not a single retry, impressive!

File: apps/explorer/src/App.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ dayjs.updateLocale('en', {
   }
 })
 
-export const numberOfAPIRetries = 10
+export const numberOfAPIRetries = 3
```
Comment (nop33) [inline apps/explorer/src/App.tsx:65]: Let's add a console.error when the threshold is reached

**Review by nop33 — APPROVED** [review]

**Pull request merged**